docs/guides/validation/how-to-add-a-validation-operator.md

```diff
@@ -0,0 +1,89 @@
+     my_second_validation_operator:
+       class_name: ActionListValidationOperator
+       action_list:
+       - name: store_validation_result
+         action:
+           class_name: StoreValidationResultAction
+     ```
+
+
+#. **Find the validation_operators section in your great_expectations config file.**
+
+   Open your project's ``great_expectations.yml`` configuration file and navigate to the ``validation_operators`` section (line 1 in the snippet). This section contains the Validation Operator instance ``action_list_operator`` that was automatically created by the ``great_expectations init`` CLI command.
+
+   |
+#. **Add a new Validation Operator block.**
+
+   Add a new block after the existing Validation Operator instances. The name of the block is the name you are giving to the new Validation Operator instance (line 15 in the snippet). These names must be unique within a project.
+
+   |
+#. **Pick a class_name.**
+
+   Add a ``class_name`` attribute in the new block you added in the previous step (line 16 in the snippet). The value is the name of the class that implements the Validation Operator that you are adding. This can be one of the classes that are included in Great Expectations or a class that you implemented. This example adds another instance of :py:class:`great_expectations.validation_operators.validation_operators.ActionListValidationOperator`.
+
+     :::note Note:
+
+       - If you are adding a custom Validation Operator, you will have to add a ``module_name`` attribute in addition to ``class_name``. You will find more details about custom Validation Operators in this :ref:`guide <how_to_guides__validation__how_to_implement_a_custom_validation_operator>`.
+
+
+#. **Configure additional fields.**
+
+   Consult the reference documentation of the class that implements the Validation Operator you are adding for additional properties (required or optional) that are specific to that class. The snippet above configured one such property specific to the :py:class:`~great_expectations.validation_operators.ActionListValidationOperator` class.
+
+#. **Test your configuration.**
+
+   Test that your new Validation Operator is configured correctly:
+
+      1. Open the configuration file of a Checkpoint you created earlier and replace the value of ``validation_operator_name`` with the value from Step 2 above. The details of Checkpoint configuration can be found in [How to add validations data or suites to a Checkpoint](../../../guides/validation/checkpoints/how-to-add-validations-data-or-suites-to-a-checkpoint).
+      2. Run the Checkpoint and verify that no errors are thrown. You can run the Checkpoint from the CLI as explained in [How to run a Checkpoint in terminal](../guides/validation/checkpoints/how-to-run-a-checkpoint-in-terminal) or from Python, as explained in [How to run a Checkpoint in python](./checkpoints/how-to-run-a-checkpoint-in-python).
+
+
+Additional notes
```

Comment:
Additional notes should not be part of the YAML but its own section right? I think we need to add closing tags to the above YAML block.